src/test/java/core/basesyntax/service/impl/FruitServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service.impl;
+
+import static core.basesyntax.model.FruitTransaction.Operation.BALANCE;
+import static core.basesyntax.model.FruitTransaction.Operation.PURCHASE;
+import static core.basesyntax.model.FruitTransaction.Operation.RETURN;
+import static core.basesyntax.model.FruitTransaction.Operation.SUPPLY;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.strategy.FruitStrategy;
+import core.basesyntax.strategy.impl.BalanceOperation;
+import core.basesyntax.strategy.impl.FruitStrategyImpl;
+import core.basesyntax.strategy.impl.PurchaseOperation;
+import core.basesyntax.strategy.impl.ReturnOperation;
+import core.basesyntax.strategy.impl.SupplyOperation;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FruitServiceImplTest {
+    private static final String BANANA = "banana";
+    private static FruitService fruitService;
+    private static FruitStrategy fruitStrategy;
+
+    @BeforeAll
+    static void beforeAll() {
+        fruitService = new FruitServiceImpl();
+        fruitStrategy = new FruitStrategyImpl(Map.of(
+                BALANCE, new BalanceOperation(),
+                RETURN, new ReturnOperation(),
+                PURCHASE, new PurchaseOperation(),
+                SUPPLY, new SupplyOperation()
+        ));
+    }
+
+    @Test
+    void getAllOperations_ok() {
```

Comment:
rename according test naming convention. the service method will be renamed as as mention in service class comments. 
`<methodUnderTest>_<state>_<expectedBehavior>;` For example, if we are testing the method register with a null
user's age the test method name should be `register_nullAge_notOk`